src/azure-cli/azure/cli/command_modules/acr/agentpool.py

```diff
@@ -43,10 +43,10 @@ def acr_agentpool_create(cmd,
     )
 
     try:
-        return client.create(resource_group_name=resource_group_name,
-                             registry_name=registry_name,
-                             agent_pool_name=agent_pool_name,
-                             agent_pool=agentpool_create_parameters)
+        return client.begin_create(resource_group_name=resource_group_name,
+                                   registry_name=registry_name,
+                                   agent_pool_name=agent_pool_name,
+                                   agent_pool=agentpool_create_parameters)
```

Comment:
what is the difference between the client previous methods with the ones with  the prefix "begin_">

Reply:
Yeah +1 if previous create waits for create to complete, does begin_create do the same or just kick of operation without completing?

Reply:
Refer to [LRP method names](https://devblogs.microsoft.com/azure-sdk/migrating-python-management-libraries/#long-running-operations-lro-method-names)

Reply:
Just name change, no behavior change.